packages/web/src/components/molecules/ExecutionStepsList/ExecutionStepsList.tsx

```diff
@@ -68,10 +68,9 @@ const ExecutionStepsList: FC<ExecutionStepsListProps> = props => {
           })
           .map(({status, icon, name, url}, index) => (
             <ExecutionStepsListItemExecution
-              // eslint-disable-next-line react/no-array-index-key
-              key={`item-${index}`}
+              key={url}
               className={classNames({clickable: url})}
-              onClick={url ? () => navigate(url) : undefined}
+              href={url ? `${baseUrl}${url}` : '#'}
```

Comment:
Either `NavLink` should be used, or additional `onClick` - otherwise, it will refresh the page on click instead of navigating as Single Page Application